integration_tests/index.d.ts

```diff
@@ -0,0 +1,29 @@
+declare namespace Cypress {
+  interface Chainable {
+    /**
```

Comment:
The `login` definition is crucial where as the `task` versions have very low value since Cypress has a generic definition, so the IDE will give a little bit of help if the code matches the definition exactly, but no real type checking is done

Reply:
ah, yes that's annoying